src/java/org/apache/cassandra/tools/nodetool/stats/TableStatsPrinter.java

```diff
@@ -167,6 +168,11 @@ protected void printStatsTable(StatsTable table, String tableDisplayName, String
             }
             out.println("");
         }
+
+        protected String formatMemory(long bytes, boolean humanReadable)
```

Comment:
@zaaath I think this should be called `formatDataSize` or similar, rather than _memory_.

Reply:
Also, could you please take a look at this? (this is an opportunistic fix here)

`out.printf(indent + "  %-" + maxWidth + "s %s%n", "Key", "Size");` 

my IDE tells me that this should be used will all `%s` rather than using concatenating. How is it now does not make a lot of sense if we use `+` together with `%s`. Should be all on `%s`. There is similar case with 

`out.printf(indent + "  %-" + maxWidth + "s %s%n", "Key", "Count");` 

below.

Reply:
@smiklosovic good point, I renamed it to `formatDataSize` and made it private.

Regarding `out.printf(indent...`, it looks weird and convoluted, but it actually makes sense. `maxWidth` sets the width of the "Key" column with left alignment which is what `%-{maxWidth}` is for. For example, let's assume that `index = ""` (empty) and `maxWidth = 10`, then the line becomes:
`out.printf("  %-10s %s%n", "Key", "Size");`
which prints:
`  Key        Size`
where the distance between "K" and "S" is `maxWidth+1=10+1=11` spaces.